google-ads/build.gradle

```diff
@@ -29,10 +29,42 @@ plugins {
     id 'com.google.api-ads.java-conventions'
     id 'com.google.protobuf' version '0.8.15'
     id "com.github.hierynomus.license-report" version "0.16.1"
+    id 'org.jreleaser' version '1.18.0'
 }
 
 description = 'Google Ads API client library for Java'
 
+jreleaser {
+    project {
+        gitRootSearch = true // Allow JReleaser to search upwards for .git
+    }
+    signing {
+        active = 'ALWAYS'
+        armored = true
+    }
+    deploy {
+        maven {
+            mavenCentral {
+                sonatype {
+                    /** Warning: this task will auto-publish artifacts to maven
+                     without an UI intervention. Publishing to Maven cannot
```

Comment:
nit: This comment is a little confusing because it seems like a general comment about the plugin, but in our case we're setting stage to UPLOAD. Consider rewording to something like "Sets stage to UPLOAD to prevent...".